ed448/src/curve/edwards/extended.rs

```diff
@@ -755,7 +756,7 @@ impl EdwardsPoint {
     /// see <https://datatracker.ietf.org/doc/rfc9380/>
     pub fn hash<X>(msg: &[u8], dst: &[u8]) -> Self
     where
-        X: for<'a> ExpandMsg<'a>,
+        X: ExpandMsg<U32>,
```

Comment:
cc @daxpedda 

I would have expected to use U84, but I'm not sure I understand why that doesn't work.

Reply:
hence the `CollisionResistance` relationship